python/rmm/tests/test_cython.py

```diff
@@ -29,7 +29,7 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
-cython_test_modules = ["rmm.tests.test_device_buffer"]
+cython_test_modules = ["rmm._test_cython.test_device_buffer"]
```

Comment:
I think maybe the better option is just to put `test_device_buffer.pyx` in a `rmm._lib.tests` subpackage rather than introduce a higher-level `rmm._test_cython` package.

Reply:
Done 👍